kong/plugins/rate-limiting/handler.lua

```diff
@@ -67,6 +67,9 @@ local function get_identifier(conf)
 
   elseif conf.limit_by == "header" then
     identifier = kong.request.get_header(conf.header_name)
+
+  elseif conf.limit_by == "path" then
+    identifier = kong.request.get_path(conf.path)
```

Comment:
`kong.request.get_path` does not expect any arguments, so `conf.path` is not used at all here.